meetings/2023-11-15.md

```diff
@@ -0,0 +1,120 @@
+* eP: This seems like a pragmatic approach to consider. There has been movement to make Solid _not_ a direct successor of LDP
+* ...: A good step might be to document that prior work and how Solid differs from LDP
+* ...: Then would it be reasonable to call it LDP 2.0
+* HZ: this proposal would make solid completely different as it takes away the operational aspect of Solid
+* PA: the goal is not to reduce Solid to reduce it to LDP 2.0, but rather to deliver one building block that can be used to build Solid, which would address some of the issues from the AC reviewers
+* HZ: I'm looking at it from the point of view of adoption. Companies are expected to make investement in this new way of building apps. The way the content is deliveered, just looking at LDP, this will take much longer. The companies will less likely invest in adoption if we limit the scope this way.
+* PA: I also wanted to respond to Solid not being strictly based on LDP -- that is something that I had missed, and so we would need to address that. There is a tension between bringing Solid as a whole to the W3C as a WG. That seems unlikely at present. Some of the concerns need to be addressed, and one way would be to reduce our ambitions to bring this to a standard track. 
+* AC: It is a very interesting idea, I need to think about it a little bit more. On the high level what I'm hearing is to make the decision, weather we are trying to specify Solid as a whole vs. specify something that would sit underneeth Solid. We can call it LDP 2.0 or something else. If we can build solid on top of that, others could try building something else on top of that. This would provide us a possibility to build solid in a meanigful way. Historically solid comes from LDP, it doesn't have a strict requirement. If LDP would be more clear on some points, and more flexible on other points we could build solid on top of that. If we were to consider something that would be LDP like I would have some ideas where we would like that to go. I like the idea of specifying somehing that solid sits on top of. This way the Solid could be very thin adding only what is missing in that base.
+* PA: This is something that has crossed my mind: people could build solid and non-solid things on top of this protocol. "We are solving a problem that is important to the Solid community, but which can also be used elsewhere". The CG could then continue to incubate client-to-client standards. Anchoring the new working group on something existing, such as LDP, would give more standing to this proposal and to continue on the existing path of existing standards. This is a pragmatic tactic. If Solid doesn't need everything on LDP, there may be parts of LDP 1.0 that could be marked deprecated or archaic; or there could be different compliance levels or profiles. Implementations might not be required to implement everything because of the different compliance levels.
```

Comment:
```suggestion
* PA: This is something that has crossed my mind: people could build Solid and non-Solid things on top of this protocol. "We are solving a problem that is important to the Solid community, but which solution can also be used elsewhere". The CG could then continue to incubate client-to-client standards. Anchoring the new working group on something existing, such as LDP, would give more standing to this proposal and to continue on the existing path of existing standards. This is a pragmatic tactic. If Solid doesn't need everything of LDP, there may be parts of LDP 1.0 that could be marked deprecated or archaic; or there could be different compliance levels or profiles. Implementations might not be required to implement everything because of the different compliance levels.
```